### PR TITLE
Add XDCR memory usage per replication to dashboard

### PR DESCRIPTION
**Pull request by nelio2k**

### CODE REVIEW

File: dashboards/xdcr-dashboard.json
Comment (dave-finlay) [inline dashboards/xdcr-dashboard.json:660]: mode of "multi" and type of "timeseries" are the default options and so are not needed if you'd like to remove them.

File: dashboards/xdcr-dashboard.json
Comment (dave-finlay) [inline dashboards/xdcr-dashboard.json:693]: Same comment as above.

**Review by dave-finlay — APPROVED** [review]
Basically fine. Tiny bit of redundant config in the panels if you'd like to remove it and push an updated patch. (I'll approve and merge right away) If you don't care about it you can feel free to just merge yourself. :-)

File: dashboards/xdcr-dashboard.json
Comment (dave-finlay) [reply dashboards/xdcr-dashboard.json:693]: Whoops - erroneously included the unit which is incorrect because you should specify "bytes". ("short" is the default). But "timeseries" and mode of "multi" aren't needed.

**Comment by nelio2k** [comment]
Did a force push to squash code review changes

**Review by dave-finlay — APPROVED** [review]

**Pull request merged**
